components/google_sheets/google_sheets.app.js

```diff
@@ -304,7 +305,7 @@ module.exports = {
       return (await sheets.spreadsheets.values.clear(request)).data;
     },
     async addRowsToSheet({
-      spreadsheetId, range, rows,
+      spreadsheetId, range, rows, params,
     }) {
       const resp = await axios({
         method: "POST",
```

Comment:
Use the `googleapis` client library instead of `axios` for making the API request in `addRowsToSheet` to [reduce code and improve maintenance](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)?